packages/grafana-ui/src/components/Table/SparklineCell.tsx

```diff
@@ -40,6 +40,20 @@ export const SparklineCell = (props: TableCellProps) => {
     );
   }
 
+  // Get the step from the first two values to null-fill the x-axis based on timerange
+  if (sparkline.x && sparkline.x.values.length > 1) {
+    sparkline.x.config.interval = sparkline.x.values[1] - sparkline.x.values[0];
+  }
+
+  // Remove non-finite values, e.g: NaN, +/-Infinity
+  sparkline.y.values = sparkline.y.values.map((v, i) => {
+    if (!isFinite(v)) {
```

Comment:
Yo! I didn't even know about `isFinite()`; nice!

Reply:
you should use `Number.isFinite()` tho ;)

Reply:
Ah, due to the implicit type conversion of non-Numbers? Nice catch. This guy JavaScripts.